test/test_verify.py

```diff
@@ -109,33 +109,33 @@ def verifyWaitTime(self):
                   self.VERIFY_INTERVAL), flush=True)
         sleep(self.VERIFY_INTERVAL)
 
-    def verifyWithRetry(self, func, round):
+    def verifyWithRetry(self, func, round, configFileName):
```

Comment:
Can we add a test to verify both should work with and without the downscoped token enabled?

Reply:
I cant think of an easier way to do it over a public internet. The only way I know is to enable/disable it is do it from snowflake account and run this tests again. Or, just set up another user for our tests and make it enable so we can run two tests in parallel. It's a lot of plumbing and I don't think it worth the efforts. 
https://github.com/snowflakedb/snowflake-kafka-connector/pull/634#issuecomment-1553576742